src/components/account/profile/profile.js

```diff
@@ -18,43 +18,48 @@ import {
 } from "@material-ui/core";
 
 const AccountProfile = props => {
-  const { className, ...rest } = props;
+  const { className, user, ...rest } = props;
```

Comment:
Why not place the { className, user, ...rest } in the function parameters, once props is not used in the component?